zachang/HelloBooks
# #152880964 admin views all borrowed books

What does this PR do?
* It allows the admin to view all borrowed books.

Description of Task to be completed?
* A page is created where the admin can view all borrowed books.

How should this be manually tested?
+ Admin should login, then click the `borrowed books` link to view all borrowed books.

Any background context you want to provide?
+ none for now.

What are the relevant pivotal tracker stories?
`#152880964`

Screenshots
![image](https://user-images.githubusercontent.com/23665997/32854113-dfe0e586-ca3d-11e7-8284-0dc867373e5f.png)

Questions:
+ none for now
